Makefile

```diff
@@ -81,14 +100,8 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
-fmt: ## Run go fmt against code.
```

Comment:
Don't you like to have it?

Reply:
We already have `golangci-lint` complaining about that with `gofmt`, IIRC.